### DISCUSSION TITLE
Tik Tok无法关注，杀后台黑屏无连接

### DISCUSSION THREAD

**Discussion by Somnr123** [Semporia/TikTok-Unlock#329] [Q&A]
圈叉解锁后，TikTok无法关注作者，观看评论都正常，怎么回事，另外同配置放在备用机11上，就变成一次性了，杀后台就黑屏了，刷不出来视频了，无网络连接，怎么解决。

**Comment by Ledrong** [comment]
你好，我是來自俄羅斯的學生。我想在https://buff.163.com/平台進行交易。但為此我需要驗證我的帳戶，而這只能使用支付寶和steam來完成。請您只需要驗證一下，一切都是安全的。我會付1000元
